### ISSUE TITLE
[Doc 2.5.0] Wrong link address in doc

### ISSUE THREAD

**Issue by archfish** [apache/pulsar#6114]
**Describe the bug**

Wrong link address in doc. doc version 2.5.

**To Reproduce**
1. Go to [rest-api](https://pulsar.apache.org/docs/en/admin-api-overview/#rest-api)
2. Click on 'document'
3. Redirect to https://pulsar.incubator.apache.org/en/admin-rest-api#/
4. Blank page

**Expected behavior**

should redirect to a valid page.

**Screenshots**
![image](https://user-images.githubusercontent.com/5317064/72788387-cce64900-3c6c-11ea-9ef9-a9a47f1d5cf9.png)
![image](https://user-images.githubusercontent.com/5317064/72788400-d4a5ed80-3c6c-11ea-9aec-ecf94259d1c7.png)

**Desktop (please complete the following information):**
 - OS:  MacOS 10.15  
 - Google Chrome

**Additional context**

use grep command find more domain `pulsar.incubator.apache.org`

```shell
grep -iR "pulsar.incubator.apache.org" .
```

**Comment by archfish** [comment]
ping @sijie

**Comment by tuteng** [comment]
The reason for this problem is that the swagger file for 2.5.0 https://github.com/apache/pulsar/tree/master/site2/website/static/swagger was not generated correctly. Are you interested in fixing this problem, please refer to the following steps:

```
git clone https://github.com/apache/pulsar
mvn clean install -DskipTests
git checkout branch-2.5
mvn -am -pl pulsar-broker install -DskipTests -Pswagger
git checkout master
git checkout -b fix/swagger-file-not-generate-for-version-2.5.0
mkdir -p site2/website/static/swagger/2.5.0
cp pulsar-broker/target/docs/*.json site2/website/static/swagger/2.5.0
```

And then, you can try new a pull request. @archfish
